scripts/setup_external_accounts.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+# Copyright 2021 Google LLC.
```

Comment:
Can the standard Apache license in the other files be applied here, or is this licensed differently intentionally?

```

Reply:
Done